### PR TITLE
Remove deprecated pkg_resources

### PR DESCRIPTION
**Pull request by marmarta**
references QubesOS/qubes-issues#9195

### CODE REVIEW

File: disk_password_gui/disk_password_gui.py [hunk 1]
```diff
@@ -148,8 +160,10 @@ def perform_setup(self):
         # pylint: disable=attribute-defined-outside-init
         self.builder = Gtk.Builder()
 
-        self.builder.add_from_file(pkg_resources.resource_filename(
-            'disk_password_gui', 'disk_password_gui.glade'))
+        glade_ref = importlib.resources.files(
+            __name__).joinpath('disk_password_gui.glade')
```
Comment (marmarek) [inline disk_password_gui/disk_password_gui.py:164]: This fails for me with `TypeError: 'disk_password_gui.disk_password_gui' is not a package`

**Review by marmarek — CHANGES_REQUESTED** [review]

**Comment by marmarta** [comment]
PipelineRefresh

**Comment by marmarta** [comment]
PipelineRetry

**Comment by marmarta** [comment]
PipelineRetry

**Pull request closed without merging**